### DISCUSSION TITLE
Building for Android locally

### DISCUSSION THREAD

**Discussion by mcbloodbug** [icosa-gallery/open-brush#766] [Q&A]
Have your friends encountered similar situations? I have PICO4 and Oculus Quest2, and I downloaded the OpenBrush source code myself for learning and modification. After packaging, I encountered the following situation:
Equipment: PICO4
Open software: Openbrush cannot be opened regardless of whether it is connected to the internet or not.
Device: Oculus Quest2
Open software: Openbrush cannot be opened regardless of whether it is connected to the internet or not.

**Comment by mikeskydev** [comment]
Hi, if you are building for yourself you probably need to change the Android package name from foundation.icosa.openbrush to something else, as it will be triggering Pico's anti-piracy protection if it's not signed with our signing key.

Edit: We see your support email too, but we'll reply here for anyone else who may be encountering this!
